lib/LoginStrategy.ts

```diff
@@ -1,56 +1,71 @@
-import { BaseStrategy, StrategyOptions } from './BaseStrategy'
-import { Request } from 'express'
-import * as jwt from 'jsonwebtoken'
-import { providers } from 'ethers'
 import {
-  ethrReg,
-  Resolver,
-  VoltaAddress1056,
-} from '@ew-did-registry/did-ethr-resolver'
+  lookup,
+  namehash,
+  areLinked,
+} from './utils'
 
-import { lookup, namehash, verifyClaim } from './utils'
 import {
   Claim,
-  DecodedToken,
   IRole,
-  IRoleDefinition,
+  DecodedToken,
   ITokenPayload,
+  IRoleDefinition
 } from './LoginStrategy.types'
-import { PublicResolverFactory } from '../ethers/PublicResolverFactory'
-import { PublicResolver } from '../ethers/PublicResolver'
+
+import {
+  ethrReg,
+  Resolver,
+  VoltaAddress1056,
+} from '@ew-did-registry/did-ethr-resolver'
+
+import { verificationMethods } from '@ew-did-registry/jwt/dist/verify'
+import { JWT } from '@ew-did-registry/jwt'
```

Comment:
yes it's true. In fact I should have put my PR as a draft, since I normally need the JWT class to use one of its verify functions. But I was wondering if I could avoid creating a hole instance and just use directly the desired function, and I went exploring this second path. I'll clean all that.